packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue

```diff
@@ -170,6 +170,7 @@ export default {
     align-items: center;
     padding: 0;
     margin: 0;
+    white-space: nowrap;
```

Comment:
is this going to potentially cause any new issues when in tight space? Also would `flex-wrap: nowrap` be useful here?

Reply:
The only issue would be that a there might be a very slight overlap, but this issue seems to only appear on the lowest size of the list view and only if there is a new tag or a local legends tag

Reply:
`flex-wrap: nowrap;` does not appear to fix the issue